cmd/get.go

```diff
@@ -6,8 +6,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	team string
```

Comment:
I'm not a big fan of global state (even for flags). I know cobra encourages this pattern, though.

Reply:
Not a fan either but these are limited to the cmd package and aren't exported. There is the possibility of adding them into the commands but that requires calling flag.Parse() in ever command function .